packages/backend/tests/unit/cooldown/models/cooldown.spec.ts

```diff
@@ -12,41 +11,24 @@ import {
 import { CooldownManager } from '../../../../src/cooldown/models/cooldownManager';
 import { bboxToWktPolygon, hashValue } from '../../../../src/common/util';
 import { HALF_GLOBE_BBOX } from '../../../../src/cooldown/models/constants';
+import redisMock from '../../../mocks/cooldown';
+import { RedisClient } from '../../../../src/redis';
 
 const NOW_MOCK = 1000;
 
-const executeIsolatedMock = jest.fn();
-const multiMock = jest.fn();
-const expireMock = jest.fn();
-const execMock = jest.fn();
-const searchMock = jest.fn();
-const setMock = jest.fn();
-
-// eslint-disable-next-line @typescript-eslint/no-unsafe-return
-jest.mock('redis', () => ({
-  ...jest.requireActual('redis'),
-  createClient: jest.fn().mockImplementation(() => ({
-    executeIsolated: executeIsolatedMock,
-    multi: multiMock,
-    expire: expireMock,
-    exec: execMock,
-    json: {
-      set: setMock,
-    },
-    ft: {
-      search: searchMock,
-    },
-  })),
-}));
-
-type RedisClient = ReturnType<typeof createClient>;
+const searchMock = redisMock.searchMock;
+const executeIsolatedMock = redisMock.executeIsolatedMock;
+const multiMock = redisMock.multiMock;
+const setMock = redisMock.setMock;
+const expireMock = redisMock.expireMock;
+const execMock = redisMock.execMock;
 
 describe('CooldownManager', () => {
   let cooldownManager: CooldownManager;
   let mockedRedis: jest.Mocked<RedisClient>;
 
   beforeAll(() => {
-    mockedRedis = createClient({}) as jest.Mocked<RedisClient>;
+    mockedRedis = redisMock.mockRedisClient as unknown as jest.Mocked<RedisClient>;
```

Comment:
declare the imported mock as `jest.Mocked<RedisClient> in the mocks dir instead of casting here, do this for all 3 mocks